src/client.rs

```diff
@@ -358,24 +310,26 @@ impl Web3 {
         }
     }
 
-    pub async fn xdai_get_latest_block(&self) -> Result<ConciseXdaiBlock, Web3Error> {
+    /// Gets the latest (non finalized) block including full tx data
+    pub async fn eth_get_latest_block_full(&self) -> Result<Block, Web3Error> {
         match self.eth_syncing().await? {
             false => {
                 self.jsonrpc_client
-                    .request_method("eth_getBlockByNumber", ("latest", false), self.timeout)
+                    .request_method("eth_getBlockByNumber", ("latest", true), self.timeout)
```

Comment:
Wondering what this boolean "true" value does to the request, these params are not well documented. Does this change the request to ask for a full block when true?